src/consumer.rs

```diff
@@ -52,7 +85,30 @@ impl PartitionConsumer {
             ))),
         })
     }
+
+    #[wasm_bindgen(js_name = "streamWithConfig")]
+    pub async fn stream_with_config(
```

Comment:
woah, this works?

Reply:
Wait, nvm, this takes ownership of `self`. I thought it was `&self`.

Reply:
Yeah 😂 why?